flutter/test/load_contexts_integrations_test.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:sentry/sentry.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+import 'mocks.dart';
+
+class MockTransport extends Mock implements Transport {}
```

Comment:
a file called `mocks.dart` already exists, what's about keeping it in there?

Reply:
✅ thanks